src/main/java/uk/gov/hmcts/reform/lrdapi/controllers/constants/LocationRefConstants.java

```diff
@@ -7,4 +7,5 @@ private LocationRefConstants() {
 
     public static final String ALL = "ALL";
     public static final String COMMA = ",";
+    public static final String EMPTY_STR = "";
```

Comment:
you will find EMPTY from StrinUtild I think

Reply:
I think even Comma also you may find in Utils

Reply:
No, `StringUtils` only contains EMPTY. I have updated it.